### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by mukul-seagate11**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @indrajitzagade @ajaysrivas @azheregelya
+*       @akash2144
```
Comment (azheregelya) [inline CODEOWNERS:5]: @mukul-seagate11, are you sure?

**Comment by azheregelya** [comment]
@mukul-seagate11 could you please add description why these changes are needed?

**Comment by mukul-seagate11** [comment]
> @mukul-seagate11 could you please add description why these changes are needed?

Its actually requested by Neerav to add Rahul as code owner for corx-ha to review the code as he will be working as developer

**Comment by ajaysrivas** [comment]
@mukul-seagate11 who is Rahul?
@NeeravChoudhari Please review what is being done here.

**Comment by NeeravChoudhari** [comment]
> @mukul-seagate11 who is Rahul?
> @NeeravChoudhari Please review what is being done here.

@mukul-seagate11  My apologies. only Read only access is required for Akash. I am correcting the resending the invite.

**Comment by mukul-seagate11** [comment]
> > @mukul-seagate11 who is Rahul?
> > @NeeravChoudhari Please review what is being done here.
> 
> @mukul-seagate11 My apologies. Read only access is required for Akash. I am correcting the request in the email as well.

~Correction, its Akash

**Pull request closed without merging**